using_openshift/managing_nodes.adoc

```diff
@@ -0,0 +1,128 @@
+
+----
+$ osc get nodes
+NAME                 LABELS              STATUS
+node1.example.com    <none>              Ready
+node2.example.com    <none>              NotReady
+----
+
+To list information about a single node, run `osc get node [replaceable]#Node_Name#`. For example:
+
+----
+$ osc get node node1.example.com
+NAME                 LABELS              STATUS
+node2.example.com    <none>              NotReady
+----
+
+The `STATUS` column of this output indicates whether the node is passing health checks from the master. Nodes can be shown in this column with the following conditions:
+
+.Node Conditions
+[cols="3a,8a",options="header"]
+|===
+
+|Condition |Description
+
+|`Ready`
+|The node has returned `StatusOK` for the health check.
+
+|`NotReady`
+|The health check is not passing.
+|===
+
+NOTE: The `STATUS` column can also show `Unknown` for a node if the CLI cannot find any node condition.
+
+You can also get more detailed information, including the reason for the current condition, by using the `osc describe node` command for a given [replaceable]#Node_Name#:
+
+[options="nowrap"]
+----
+$ osc describe node node2.example.com
+Name:	node2.example.com
+Conditions:
+Kind    Status  LastProbeTime                   LastTransitionTime              Reason                                                              Message
+Ready   None    Tue, 03 Mar 2015 22:53:22 +0000 Tue, 03 Mar 2015 21:27:50 +0000 Node health check failed: kubelet /healthz endpoint returns not ok
+----
+
+== Adding a Node
+To add a node to an existing OpenShift instance:
+
+. Create a file with the following JSON format:
++
+.[filename]#node.json# file
+====
+
+----
+{
+  "id": "node3.example.com", <1>
+  "kind": "Node", <2>
+  "apiVersion": "v1beta1", <3>
+  "resources": { <4>
+    "capacity": {
+      "cpu": 1,
+      "memory": 1073741824
+    },
+  },
+  "labels": { <5>
+    "name": "node3",
+  },
+}
+----
+
+<1> Set [parameter]#id# to the fully-qualified domain name where the node can be reached. This value can be seen in the `NAME` column of `osc get nodes` output.
+<2> Set [parameter]#kind# to `Node` to let Kubernetes know this is a manifest for a node object.
+<3> Set [parameter]#apiVersion# to the Kubernetes API version you are using.
+<4> Set any resources that should be associated with the node.
+<5> Set any desired labels.
+====
+
+. Use the `osc create` command to create the node object using the file you created:
++
+****
+$ `osc create -f [replaceable]#node.json#`
+****
++
+At this point, the master begins validating the new node with health checks.
+
+. Verify that the node was added:
++
+----
+$ osc get nodes
+NAME                 LABELS              STATUS
+node1.example.com    <none>              Ready
+node2.example.com    <none>              Ready
+node3.example.com    <none>              Ready
+----
++
+Pods are not scheduled for the node until it passes the master's health checks and is placed in the `Ready` condition.
+
```

Comment:
[suggestion]
Pods are not scheduled for the node until the node passes the health checks that are performed by the master. When the node passes these health checks it is then placed in the `Ready` condition.